src/chainstate/stacks/index/node.rs

```diff
@@ -569,18 +593,17 @@ impl TrieCursor {
 pub struct TrieLeaf {
     pub path: Vec<u8>,      // path to be lazily expanded
     pub data: MARFValue,    // the actual data
-    backptr: TriePtr        // pointer back to the previous version of this leaf
 }
 
 impl fmt::Debug for TrieLeaf {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        write!(f, "TrieLeaf(path={} data={} backptr={:?})", &to_hex(&self.path), &to_hex(&self.data.to_vec()), &self.backptr)
+        write!(f, "TrieLeaf(path={} data={})", &to_hex(&self.path), &to_hex(&self.data.to_vec()))
```

Comment:
Please restore the `backptr` information -- it's helpful when debugging.

Reply:
I don’t think it was being set — but I can add that back in.

Reply:
Okay -- I think that the conclusion from the blockchain meeting was that the TrieLeaf back pointer field wasn't being used (and also shouldn't be in the future), so I can leave the back pointer removed from the TrieLeaf struct.